src/kontrol/kdist/cheatcodes.md

```diff
@@ -378,20 +378,63 @@ This rule returns a symbolic integer of up to the bit width that was sent as an
        [preserves-definedness]
 ```
 
+#### `randomUint` - Returns a single symbolic unsigned integer of a given size.
+
+```
+function randomUint() external returns (uint256);
+function randomUint(uint256) external returns (uint256);
+function randomUint(uint256,uint256) external returns (uint256);
+```
+
+`cheatcode.call.randomUint` will match when the `randomUint` cheat code function is called.
+This rule returns a symbolic integer of up to the bit width that was sent as an argument.
+
+```{.k .symbolic}
+    rule [cheatcode.call.randomUint]:
```

Comment:
The naming of the rule seems a bit confusing here, as this cheat code is the one that takes the width argument.

Reply:
Thanks! Fixed in https://github.com/runtimeverification/kontrol/pull/877/commits/226e3f5fa9c94e60986c7d945a69caffb52aca16.